CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+- Keep your branch up-to-date :
+```
+git checkout <your_branch>
+git fetch
+git rebase origin/dev
+```
+
+### Bugs
+
+The way support is delivered greatly impacts how users perceive the product and the team.
+Handling the incoming support requests efficiently is thus a top priority.
+
+- Bug must have a correct description and the bug's issue template must be filled carefully.
+- Bug must be tagged with:
+  - `kind:bug`
+  - one or several `area:<affected_area>` if possible, if the affected area is not known leave it blank it will be added later by another team member`.
+  - one `severity:<bug_severity>` if possible, if severity is not known leave it blank it will be added later by another team member
+- Contributors can change issues' tags (severity, area, kind, ...) if necessary. 
+  Keep in mind that explaining why you are editing an issue is the best way to avoid misunderstanding.
+- If you are working on a bug or plan to work on a bug, assign yourself to the bug.
+  If a bug is assigned, consider that they are already working on the resolution.
+- PRs solving bugs should add a regression tests to ensure that bug will not be back in the future.
+
+### General code recommendations
+
+- Take the time to pick good names.
+  Avoid non well-known abbreviations.
+
+### Python
+
+- Follow [the Zen of Python](https://www.python.org/dev/peps/pep-0020/).
+- To keep our API surface minimal, only make public what is necessary.
+- Documentation should only be there to explain things that aren't self evident.
+- Code is linted with [flake8](https://github.com/csachs/pyproject-flake8).
+- Code is formatted with [Black](https://github.com/psf/black).
+- Imports are sorted with [Isort](https://github.com/PyCQA/isort).
+
+### Test
+
+- Python tests are written using [pytest](https://docs.pytest.org/).
+
+### Rust
+
+- To keep our API surface minimal, only make public what is necessary.
+- Documentation should only be there to explain things that aren't self evident.
+- Use the [documentation example](https://doc.rust-lang.org/rust-by-example/meta/doc.html) to know how to phrase and format your documentation.
+- Code is linted with [clippy](https://github.com/rust-lang/rust-clippy).
+- Code is formatted with [fmt](https://github.com/rust-lang/rustfmt).
+
+### Test
+
+- Test as much code as possible to avoid future regression and show that added code is bullet proof.
+- Rust code is tested files per files following these [recommendations](https://doc.rust-lang.org/book/ch11-01-writing-tests.html).
+
+### Java
+
+- To keep our API surface minimal, only make public what is necessary.
+- All public methods must be documented.
+  Avoid paraphrasing doc.
+- Code is formatted with [checkstyle](https://checkstyle.sourceforge.io/).
+
+#### Tests
+
+- TODO
+
+### Javascript / Typescript / Front
+
+- To keep our API surface minimal, only make public what is necessary.
+- Documentation should only be there to explain things that aren't self evident.
```

Comment:
Move to the "general code recommandations" section

Reply:
same here https://github.com/DGEXSolutions/osrd/pull/3347#discussion_r1134013899